sdk/logs/src/test/java/io/opentelemetry/sdk/logs/SdkLogEmitterTest.java

```diff
@@ -34,15 +35,19 @@
   private static final String INSTRUMENTATION_LIBRARY_NAME = SdkLogEmitter.class.getName();
   private static final String INSTRUMENTATION_LIBRARY_VERSION = "0.0.1";
   private static final String SCHEMA_URL = "http://schemaurl";
+  private static final long NOW = TimeUnit.MILLISECONDS.toNanos(System.currentTimeMillis());
 
   @Mock private LogProcessor logProcessor;
+  @Mock private Clock clock;
```

Comment:
do we need a mock clock for this test? It looks like we're always assigning the time explicitly to the logs.

Reply:
Ah I didn't realize that. I'll pull it in to a new test method.